### DISCUSSION TITLE
init-hook import-error

### DISCUSSION THREAD

**Discussion by skoomer** [pytest-dev/pytest#9978] [Q&A] [answered]
Hello, I have a problem and I can't seem to solve it
when i run linter
************* Module doctorweb.apps.doctor.models
app/doctorweb/apps/doctor/models.py:3:0: E0401: Unable to import 'apps.accounts.models' (import-error)

in .pylintrc file
init-hook='import sys; sys.path.append("apps")'
app/doctorweb/config/settings.py

import sys
sys.path.append('apps')

**Comment by The-Compiler** [comment, marked as answer]
pylint is entirely unrelated to pytest.
